packages/grid/src/vaadin-grid-data-provider-mixin.js

```diff
@@ -187,18 +187,18 @@ export const DataProviderMixin = (superClass) =>
      * @param {HTMLElement} el
      * @protected
      */
-    _getItem(index, el) {
-      el.index = index;
+    _getItem(index, row) {
```

Comment:
The JSDoc above should be updated accordingly to mention `row` instead of `el`.

Reply:
Thanks, done.